package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "hello-react-front-end",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
- [ ] There are a couple of packages that are missing as dependencies in your package.json. One major one is the `React-router-dom`, `redux`, etc. See the image below for a list of missing packages. Please install them in your project, so that they appear in the package.json making it easy for any one using your repo to run the command `npm install` and all packages are installed.

![image](https://github.com/Piercel2022/hello-react-front-end/assets/84156714/7892645d-add4-47e7-9377-2cdf75b9bf5b)